flaskr/manage.py

```diff
@@ -14,7 +14,12 @@
 # Add command line arguments for database and shell with ORM context
 manager.add_command('db', MigrateCommand)
 def make_shell_context():
-    return dict(app=app, db=db, Language=Language, Method=Method)
+    return dict(app=app,
```

Comment:
assuming this "code first in action"?

Reply:
This is how the Flask book recommended setting up the shell environment so that the models are pre-loaded; feels like there should be an easier way, but it works for now.